rep-0008.rst

```diff
@@ -67,12 +133,12 @@ Module with msgs/srvs::
           |- yourfiles.py
      |- scripts/
         |- non-exported python files
-    
+
 We distinguish between "nodes" and "scripts" for clarity to
 users. Nodes are executable Python files that conform to the ROS node
 API. Scripts are executable Python files that do not conform to the
 ROS node API.
```

Comment:
@jack-oquin I agree with you, there are inconsistencies or lack of details in some of the catkin docs on this topic. But I think enumerating those instances and fixing them doesn't necessarily have to happen before we update the REP. I think of this as step one.

I've setup a new issue to track some of this, please comment there with pages you find that are problematic:

https://github.com/ros/rosdistro/issues/6946

I'll update the REP with the proposed changes soon.